backend/src/middleware/logger.middleware.ts

```diff
@@ -0,0 +1,34 @@
+import { UserService } from '@/user/user.service';
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { NextFunction, Request, Response } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  constructor(
+    private jwt: JwtService,
+    private userService: UserService
+  ) {}
+  async use(req: Request, res: Response, next: NextFunction) {
+    const cookie = req.cookies['jwt'];
+    if (!cookie) {
+      return res.status(401).json({ message: 'Unauthorized User' });
```

Comment:
use nestjs common exception https://docs.nestjs.com/exception-filters#built-in-http-exceptions